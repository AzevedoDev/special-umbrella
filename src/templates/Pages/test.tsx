import { renderWithTheme } from 'utils/tests/helpers'
import { screen } from '@testing-library/react'

import TemplatePage from '.'

const loterias = [
  {
    id: 0,
    nome: 'mega-sena'
  },
  {
    id: 1,
    nome: 'quina'
  },
  {
    id: 2,
    nome: 'lotof√°cil'
  },
  {
    id: 3,
    nome: 'lotomania'
  },
  {
    id: 4,
    nome: 'timemania'
  },
  {
    id: 5,
    nome: 'dia de sorte'
  }
]
const concurso = {
  id: '5534',
  loteria: 1,
  numeros: ['17', '32', '41', '58', '69'],
  data: '2021-04-26T21:12:23.562Z'
}
const loteriaFiltrado = { id: '1', nome: 'mega-sena' }

describe('<Select />', () => {
  it('should render and match with snapshot', () => {
    renderWithTheme(
      <TemplatePage
        concurso={concurso}
        loterias={loterias}
        loteriaFiltrado={loteriaFiltrado}
      />
    )

    expect(screen.getByDisplayValue(/mega-sena/i)).toBeInTheDocument()
    expect(screen.getByDisplayValue(/mega-sena/i)).toMatchSnapshot()
  })
})
