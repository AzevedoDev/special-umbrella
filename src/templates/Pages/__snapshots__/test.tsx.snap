// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> should render and match with snapshot 1`] = `
<select
  aria-label="select-lotteries"
>
  <option
    value="0"
  >
    MEGA-SENA
  </option>
  <option
    value="1"
  >
    QUINA
  </option>
  <option
    value="2"
  >
    LOTOF√ÅCIL
  </option>
  <option
    value="3"
  >
    LOTOMANIA
  </option>
  <option
    value="4"
  >
    TIMEMANIA
  </option>
  <option
    value="5"
  >
    DIA DE SORTE
  </option>
</select>
`
